_includes/toc.html

```diff
@@ -6,7 +6,7 @@ <h1 class="no_toc">
     </a>
 </h1>
 <div class="content is-hidden">
-1. TOC
-{:toc}
 </div>
-</div>
\ No newline at end of file
+</div>
+* auto-gen TOC:
```

Comment:
@bebatut Can we go ahead and delete this file, as it is no longer needed to display the TOC?

Reply:
Yes

Reply:
Thanks.